db/go.mod

```diff
@@ -6,4 +6,7 @@ require (
 	github.com/dgraph-io/badger/v3 v3.2103.1
 	github.com/google/btree v1.0.0
 	github.com/stretchr/testify v1.7.0
+	github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c
 )
+
+replace github.com/tecbot/gorocksdb => github.com/roysc/gorocksdb v1.1.0 // v0.0.0-20210804143633-c0bf0b3635e5 // FIXME
```

Comment:
would be good to add more context to the FIXME for the next person

Reply:
Sorry, I had hoped there might be movement on their upstream repo before this was to be merged. That seems unlikely now.

Would it be better to have this point to a `cosmos/gorocksdb` fork before merging?

Reply:
if you dont want to maintain it, this would be the best solution. @alexanderbez what you think?

Reply:
I mean I honestly don't see us maintaining it -- they would diverge quickly I imagine. I'm actually concerned that it's not merged into the main upstream repo. Is there a reason for that?

Reply:
It seems to just be neglected, there's been no activity for almost two years. The PR adds some some simple bindings. I don't really mind using my fork, just not sure what the preference is on your end.

Reply:
TBH, if a db backend has been neglected for that long, I'm very very apprehensive about supporting it. We can totally not support RocksDB.

Reply:
Just to be clear, this is an unofficial Go wrapper I'm talking about. The Rocks maintainers were pretty efficient about merging my PR to add the needed C bindings.

Reply:
I think its fine, our default db currently, leveldb, is not maintained as well. I would prefer we create a fork in cosmos and add a downstream pr opening bot if anything changes upstream.

Reply:
> The Rocks maintainers were pretty efficient about merging my PR to add the needed C bindings.

Then we should go with this. I do not feel comfortable maintaining support for physical backends that are not maintained, even if they're just wrappers.

Reply:
RocksDB doesn't provide Go bindings. The tecbot repo is the recommended third-party solution: https://github.com/facebook/rocksdb/blob/main/LANGUAGE-BINDINGS.md. Do you mean we should write a separate wrapper?